WS2012R2/lisa/setupscripts/STOR_VSS_BackupRestore_Fail.ps1

```diff
@@ -304,7 +304,7 @@ if(-not $EventLog)
 # Event ID 10107 is what we looking here, it will be always be 10107.
 foreach ($event in $EventLog)
    {
-       Write-Output "VSS Backup Error in Event Log number is $EventLog:" >> $summaryLog
+       Write-Output "VSS Backup Error in Event Log number is $($event.ID):" >> $summaryLog
        if ($event.Id -eq 10150)

```

Comment:
Hi Nick,
Which event ID will be captured on host? here are different ID checked in comments and codes(10107 and 10150), In fact, i can't capture the both ID when i test this case, please help me to check this  case

Thanks